src/d3d9/d3d9_device.cpp

```diff
@@ -1840,6 +1840,12 @@ namespace dxvk {
 
     D3D9DeviceLock lock = LockDevice();
 
+    // D3DCLEAR_ZBUFFER and D3DCLEAR_STENCIL are invalid flags
+    // if there is no currently bound DS (which can be the autoDS)
+    if (unlikely(m_state.depthStencil == nullptr
+            && ((Flags & D3DCLEAR_ZBUFFER) || (Flags & D3DCLEAR_STENCIL))))
```

Comment:
Looks weird when this can be written more concisely as `Flags & (D3DCLEAR_ZBUFFER | D3DCLEAR_STENCIL)`?

Reply:
Makes sense, of course. I have to stop writing code when I'm half asleep.